docs/UserGuide.md

```diff
@@ -163,35 +233,105 @@ Mark the specified task from the planner as done.
+
 * `IMPORTANCE` is one of **HIGH**, **MEDIUM**, **LOW**.
+
 * `ADDITIONAL-NOTES` is the additional notes of the task.
-   
-#### Example usage
 
-Use [list feature](#list-tasks-list-or-l) first to get the `INDEX` of the task to be edited.
-* `edit 1 n/Assignment2 t/1800 D/17-09-2020 d/1 r/today i/high a/Refer to lecture notes`
-* `edit 1 t/1800`
-* `e 1 n/Assignment2 t/1800 D/17-09-2020 d/1 r/today i/high a/Refer to lecture notes`
+##### Command Format
 
-#### Expected output
+`edit TASK-ID [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+
+##### Example Usage
+
+Firstly, use [list feature](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be edited.
+
+The following shows the output from ATHENA after entering `edit 1 n/Assignment2 t/1800 D/17-09-2020 d/1 r/16-09-2020 i/low a/Refer to notes`.
+You should expect to see a message to confirm that the task with index 1 is edited.
 
-For first example usage `edit 1 n/Assignment2 t/1800 D/17-09-2020 d/1 r/today i/high a/Refer to lecture notes`:
 ![edit command screenshot](screenshots/athena%20edit.jpg)
 
+#### Intermediate Usage
+
+The `edit` command accepts 8 parameters.
+
+* `TASK-ID` refers to the number shown beside the task that the user wants to edit in the displayed task list. It must be a non-negative integer.
+
+At least one parameter shown below have to be edited:
+* `NAME` is the name of the task.
+
+* `TIME` is the time to start doing this task (**HHMM**). For example, 1100.
+
+* `DURATION` is the expected time taken to complete task (in hours).
+
+* `DEADLINE` is the date to do task by (**DD-MM-YYYY**). For example, 16-09-2020.
+
+* `RECURRENCE` is one of **TODAY**, **MONDAY**, **TUESDAY**, **WEDNESDAY**, **THURSDAY**, **FRIDAY**, **SATURDAY**, **SUNDAY** or a specific date (**DD-MM-YYYY**).
+
+* `IMPORTANCE` is one of **HIGH**, **MEDIUM**, **LOW**.
+
+* `ADDITIONAL-NOTES` is the additional notes of the task.
+
+##### Command Format
+
+`edit TASK-ID [n/NAME] [t/TIME] [d/DURATION] [D/DEADLINE] [r/RECURRENCE] [i/IMPORTANCE] [a/ADDITIONAL-NOTES]`
+
+##### Example Usage
+
+Firstly, use [list feature](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be edited.
+
+The following shows the output from ATHENA after entering `edit 1 t/1800`.
+You should expect to see a message to confirm that the time of task with `TASK-ID` '1' is edited.
+
+*TODO: Insert screenshot*
+
+#### Advanced Usage
+
+This section is an extension to the [Intermediate Usage](intermediate-usage) section.
+
+The `edit` command supports shortcuts for advanced users.
+
+##### Supported Shortcuts
+
+The following commands on the left will be expanded to the corresponding commands on the right.
+
+| Shortcut | Expanded command |
+|----------|------------------|
+| `e`     | `edit`  |
+
+##### Example usage
+
+Firstly, use [list feature](#list-tasks-list-or-l) to get the `TASK-ID` of the task to be edited.
```

Comment:
I feel that advanced users already know to use list command so dont need this line ba